assets/flashcards.js

```diff
@@ -147,14 +163,6 @@ const clearStage = (callerPlaySound) => {
   hideScoreBar(scoreBar);
 };
 
-const addClearStageClickListener = () => {
-  // Button for clearing the working stage
-  const clearStageButton = document.getElementById("clearstagebutton");
-  clearStageButton.addEventListener("click", () => {
-    clearStage(true);
-  });
-};
-
```

Comment:
Moved. There's another one buried in here somewhere that you should go find. These should all be grouped together